codebase/superdarn/src.lib/tk/fitacf_v3.0/src/determinations.h

```diff
@@ -34,7 +34,10 @@ July 2015
 #include "fitblk.h"
 #include "llist.h"
 
+#ifndef _RMATH_H
 #define LN_TO_LOG 1/log(10) /*natural log to log conversion*/
+#endif
+
```

Comment:
I think this change has caused some problems in fitacf3. On this branch the values of `p_l`, `p_l_e`, `p_s` and `p_s_e` are clearly too high. After removing this change I get the same values as `develop`.

Reply:
So I cannot find those lines in the code ...

Reply:
ignore me on a different branch

Reply:
K changed